Components/GamesDisplay/index.js

```diff
@@ -0,0 +1,161 @@
+  });
+
+  const regularseasonGames = filteredGames.filter((game) => !game.postseason);
+
+  const postseasonGames = filteredGames.filter((game) => game.postseason);
+
+  const gamesWon = showPostseason
+    ? postseasonGames.filter(
+        (game) =>
+          (game.home_team.id === parseID &&
+            game.home_team_score > game.visitor_team_score) ||
+          (game.visitor_team.id === parseID &&
+            game.visitor_team_score > game.home_team_score)
+      )
+    : regularseasonGames.filter(
+        (game) =>
+          (game.home_team.id === parseID &&
+            game.home_team_score > game.visitor_team_score) ||
+          (game.visitor_team.id === parseID &&
+            game.visitor_team_score > game.home_team_score)
+      );
+
+  const gamesLost = showPostseason
+    ? postseasonGames.filter(
+        (game) =>
+          (game.home_team.id === parseID &&
+            game.home_team_score < game.visitor_team_score) ||
+          (game.visitor_team.id === parseID &&
+            game.visitor_team_score < game.home_team_score)
+      )
+    : regularseasonGames.filter(
+        (game) =>
+          (game.home_team.id === parseID &&
+            game.home_team_score < game.visitor_team_score) ||
+          (game.visitor_team.id === parseID &&
+            game.visitor_team_score < game.home_team_score)
+      );
+
+  let seasons = [];
+  for (let year = CURRENT_SEASON; year >= 1946; year--) {
+    seasons.push(year);
+  }
+  const selectYears = seasons.map((season) => ({
+    value: season,
+    label: season.toString(),
+  }));
+
+  function handleSeasonChange(selectedOption) {
+    setSeason(selectedOption.value);
+  }
+
+  function toggleView() {
+    setShowPostseason((prevState) => !prevState);
+  }
+
+  return (
+    <>
+      {filteredGames.length > 0 ? (
+        <GamesContainer>
+          <SelectionContainer>
+            <Headline>Games</Headline>
+            <label htmlFor="season-select">Season:</label>
+            <StyledSelect
+              value={selectYears.find((option) => option.value === season)}
+              options={selectYears}
+              onChange={handleSeasonChange}
+              placeholder="pick a season"
+              id="season-select"
+            />
+            <h3>{showPostseason ? "Postseason" : "Regular Season"}</h3>
+            <h4>Wins: {gamesWon.length}</h4>
+            <h4>Losses: {gamesLost.length}</h4>
+            <button type="button" onClick={toggleView}>
+              {showPostseason ? "Show Regular Season" : "Show Postseason"}
+            </button>
+          </SelectionContainer>
+          {showPostseason ? (
+            postseasonGames.length > 0 ? (
+              postseasonGames.map((game) => (
+                <GamesList key={game.id}>
+                  <SingleGame key={game.id}>
+                    {game.date.split("T")[0]}
+                    {": "}
+                    <br />
+                    <br />
```

Comment:
I think it is better to use `p` elements that you style to create the break between them or with a new list inside `SingleGame`. Otherwise you could also style a `br` and set a margin instead of using two `br`[(see MDN)](https://nextjs.org/docs/pages/building-your-application/optimizing/fonts#local-fonts). But there are some [accessibility concerns](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns) when you use `br` so I would prefer another solution. ;)